spec/integration/module_tool/tar/mini_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'puppet/module_tool'
+require 'puppet_spec/files'
+
+describe Puppet::ModuleTool::Tar::Mini, :if => (Puppet.features.minitar? and Puppet.features.zlib?) do

```

Comment:
Prefer to use `&&` instead of `and` here as the precedence rules are different.  `and` is intended to be a control flow statement (I presume this was copied from the existing test at https://github.com/puppetlabs/puppet/blob/master/spec/unit/module_tool/tar/mini_spec.rb#L4 but we should probably not copy that part)

http://stackoverflow.com/a/1426835